monitoring/statistics.cc

```diff
@@ -330,10 +330,15 @@ uint64_t StatisticsImpl::getAndResetTickerCount(uint32_t tickerType) {
 
 void StatisticsImpl::recordTick(uint32_t tickerType, uint64_t count) {
   assert(tickerType < TICKER_ENUM_MAX);
-  per_core_stats_.Access()->tickers_[tickerType].fetch_add(
-      count, std::memory_order_relaxed);
-  if (stats_ && tickerType < TICKER_ENUM_MAX) {
-    stats_->recordTick(tickerType, count);
+  if (get_stats_level() <= StatsLevel::kExceptTickers) {
+    return;
+  }
+  if (tickerType < TICKER_ENUM_MAX) {
+    per_core_stats_.Access()->tickers_[tickerType].fetch_add(
+        count, std::memory_order_relaxed);
+    if (stats_) {
+      stats_->recordTick(tickerType, count);
+    }
   }
```

Comment:
I would have removed the assertion above and used `else { assert(false); }` but that's my taste.